examples/graphiql-plugin.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const Fastify = require('fastify')
+const mercurius = require('..')
+
+const app = Fastify()
+
+const pluginSrc = `
```

Comment:
is there a chance we can provide the original source code instead of the umd precompiled one? this makes reading this code nearly impossible. why don't we provide an example which shows the whole thing?

Reply:
I move the example in a folder and I'll add the sources of the plugin with the package.json and the rollup config.

Reply:
Done